### DISCUSSION TITLE
Confusion creating containers as System owner

### DISCUSSION THREAD

**Discussion by GlitchedVortex** [cockpit-project/cockpit#18328] [Q&A]
Hi
I am kinda new to using Podman and containers in general and hoping to get some further details with what i am trying to do.
So from what i understand, the System owner is basically a root user.

I made a container as the logged in user and then went to the webpage via the ip and port and it works.
But when i set up the container in the same way with the System owner and then go to the IP and port, the page doesnt load.

From what i can understand, it's something to do with there not being a network perhaps? But i am still confused on how i can make the container work with the System owner instead of the logged in user.

Can anyone be kind enough as to provide further details on how i can make the cotainer work in the way i am wanting (Assuming it is possible) and any other details that can be of possible help to me.

Thank you

**Comment by marusak** [comment]
> I made a container as the logged in user and then went to the webpage via the ip and port and it works.
But when i set up the container in the same way with the System owner and then go to the IP and port, the page doesnt load.

You will have to provide more details, like what images you are running, details about the running containers, how you are trying to reach them...

In general there is no restriction that system containers could not be reached unless you have configured something. I have tried to run `httpd` as system container and I can reach it just fine.

**Reply by GlitchedVortex** [reply]
> > I made a container as the logged in user and then went to the webpage via the ip and port and it works.
> > But when i set up the container in the same way with the System owner and then go to the IP and port, the page doesnt load.
> 
> You will have to provide more details, like what images you are running, details about the running containers, how you are trying to reach them...
> 
> In general there is no restriction that system containers could not be reached unless you have configured something. I have tried to run `httpd` as system container and I can reach it just fine.

When i make a rootful podman container for quickchart. I set up the ip and the port as it sould be. I am able to curl the IP from the terminal in cockpit. But when i browse to that page with the ip and port, it does not load. The page just times out. 
It's almost like it's trying to connect me but i am being blocked from connecting to it. 
I attempted to open the port on the firewall and it shows to be open but this still did not help my situation at all. 
So it seems like i might be missing some kind of step somewhere?

**Reply by marusak** [reply]
can you please share the exact commands/options you use for rootless that works and for system that do not work?
